src/rpc/blockchain.cpp

```diff
@@ -790,8 +790,8 @@ static RPCHelpMan getblockfrompeer()
 {
     return RPCHelpMan{
         "getblockfrompeer",
-        "Attempt to fetch block from a given peer.\n"
-        "\nWe must have the header for this block, e.g. using submitheader.\n"
+        "Attempt to fetch block from a given peer.\n\n"
+        "We must have the header for this block, e.g. using submitheader.\n"
```

Comment:
Can we fix the `nReturns` below just for consistency?

Reply:
Yes, agreed.